src/Halogen/HTML/Events/Types.purs

```diff
@@ -33,6 +36,23 @@ type MouseEvent =
   , which :: Number
   )
 
+-- | Identifies the additional fields which are available on drag events.
+type DragEvent =
+  ( button :: Number
+  , detail :: Number
+  , relatedTarget :: HTMLElement
+  , clientX :: Number
+  , clientY :: Number
+  , screenX	:: Number
+  , screenY	:: Number
+  , ctrlKey	:: Boolean
+  , shiftKey :: Boolean
+  , altKey :: Boolean
+  , metaKey	:: Boolean
+  , which :: Number
+  , dataTransfer :: Nullable DataTransfer
```

Comment:
I think it is okay if we merge it as is, and try to come up with a nicer solution in a separate PR.

There is nothing wrong with having `Nullable` there, it's just a small inconvenience of having to use `Nullable.toMaybe`.